lib/template.js

```diff
@@ -15,6 +15,12 @@ const template = (string, obj) => {
     }
     return result
   })
+  if (customReplacers) {
+    customReplacers.forEach(({ search, replace }) => {
+      str = str.replace(new RegExp(search, 'g'), replace)
```

Comment:
I agree with the flags, i'll add it as another property.
But the / syntax makes things super difficult because shouldn't get imported into `new RegExp`. So i'd have to process and delete the slashes, and it gets complicated.

```
new RegExp('/abc/')
//abc//
new RegExp('abc')
/abc/
```

Reply:
I'd prefer not adding another property and instead suggest you look at [regex-parser](https://www.npmjs.com/package/regex-parser)

Should handle parsing the regex in both formats only concern is to take the regex you get and add the default `g` flag :sweat_smile:

Reply:
Ooooo I didn't know it existed. I always assumed parsing regex was super complicated.

Reply:
Feel free to try out regex-parser: https://npm.runkit.com/regex-parser

Reply:
```javascript
const regexParser = require("regex-parser")
let re = regexParser('/JENKINS-(\d+)/i')
if (!re.flags.includes('g')) { re = new RegExp(re.source, re.flags + 'g') }
if (re.flags.includes('g')) { console.log('yes') }
else { console.log('no') } 
console.log(re.source, re.flags)
```

Reply:
One could argue for `jenkins.com` regex you'd want to escape the `.` [escape-string-regexp](https://www.npmjs.com/package/escape-string-regexp)

Reply:
so that would force global with no way to disable it.  I can't decide my feeling on that.

Reply:
I wouldn't want to auto escape anything for the user.

Could make it so if it starts with a /, its a regex, otherwise its a string?

Reply:
If we support regex I'd be fine without auto global 😊

Reply:
The check should be does not startswith / and flags is empty for auto global

Reply:
/foo/bar/
flags is ""

Does that mean they don't want global? or want the default?
I think once you regex, you control all the flags